tests/integration/rbac_test.go

```diff
@@ -67,19 +71,48 @@ func rbacTests(t *testing.T, when spec.G, it spec.S) {
 			if err == nil {
 				t.Fatal("rio-standard should not be able to create service that enable service mesh")
 			}
-			assert.True(t, strings.Contains(err.Error(), insuffienctPrivilegesMsg))
-			assert.True(t, strings.Contains(err.Error(), "rio-servicemesh"))
+			assert.Contains(t, err.Error(), insuffienctPrivilegesMsg)
+			assert.Contains(t, err.Error(), "rio-servicemesh")
 		})
 
-		// TODO: we need to add more test for custom verb: hostport, hostnet, hostmount, serviceMesh and privilege
 		it("rio-standard should not be able to create host-network services", func() {
 			testService.Kubeconfig = standardUser.Kubeconfig
 			err := testService.CreateExpectingError(t, "--net", "host", "nginx")
 			if err == nil {
 				t.Fatal("rio-standard should not be able to create service that enable host networking")
 			}
-			assert.True(t, strings.Contains(err.Error(), insuffienctPrivilegesMsg))
-			assert.True(t, strings.Contains(err.Error(), "rio-hostnetwork"))
+			assert.Contains(t, err.Error(), insuffienctPrivilegesMsg)
+			assert.Contains(t, err.Error(), "rio-hostnetwork")
+		})
+
+		it("rio-standard should not be able to create host-port services", func() {
+			testService.Kubeconfig = standardUser.Kubeconfig
+			err := testService.CreateExpectingError(t, "-p", "80,hostport", "nginx")
+			if err == nil {
+				t.Fatal("rio-standard should not be able to create service that enable hostport")
+			}
```

Comment:
You can use `assert.Error` instead of this pattern, below too

Reply:
:+1: